source/core/index.ts

```diff
@@ -632,6 +633,7 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 		typedResponse.isFromCache = (this._nativeResponse as any).fromCache ?? false;
 		typedResponse.ip = this.ip;
 		typedResponse.retryCount = this.retryCount;
+		typedResponse.ok = (statusCode >= 200 && statusCode <= 299);
```

Comment:
https://github.com/sindresorhus/got/blob/c25af092bf792c2c7e29db848a569f2eb3e527f8/source/core/response.ts#L109-L114

Reply:
Should I check followRedirect & limit status code similar to this before assigning boolean to ok?